ui/src/alert-configuration/ConfigureAlerting.test.tsx

```diff
@@ -0,0 +1,1275 @@
+
+    return res(
+      ctx.json({
+        return: [
+          {
+            token: 'xxx-yyy-zzzz-token',
+          },
+        ],
+      }),
+    );
+  }),
+
+  rest.post('http://localhost/api/salt', (req, res, ctx) => {
+    return res(ctx.json({}));
+  }),
+
+  rest.get('http://localhost/api/kubernetes/api/v1/nodes', (req, res, ctx) => {
+    const result = {
+      items: [
+        {
+          metadata: {
+            name: 'node1',
+            labels: {
+              'metalk8s.scality.com/version': '125.0.0',
+            },
+          },
+        },
+      ],
+    };
+    return res(ctx.json(result));
+  }),
+
+  rest.get(
+    'http://localhost/api/kubernetes/api/v1/namespaces/metalk8s-monitoring/pods',
+    (req, res, ctx) => {
+      const result = {
+        items: [
+          {
+            metadata: {
+              name: 'pods',
+            },
+          },
+        ],
+      };
+      return res(ctx.json(result));
+    },
+  ),
+
+  rest.post(
+    'http://localhost/api/alertmanager/api/v1/alerts',
+    (req, res, ctx) => {
+      const result = {
+        status: 'success',
+      };
+      return res(ctx.json(result));
+    },
+  ),
+);
+
+const commonSetup = async () => {
+  render(
+    <Switch>
+      <Route exact path="/alerts">
+        <div>Redirected Alert Page</div>
+      </Route>
+      <Route exact path="/">
+        <ConfigureAlerting />
+      </Route>
+    </Switch>,
+  );
+
+  await waitForLoadingToFinish();
+};
+
+describe('<ConfigureAlerting />', () => {
+  beforeAll(() => {
+    server.listen({ onUnhandledRequest: 'error' });
+  });
+  beforeEach(() => {
+    saltLoginRequest.mockClear();
+    patchAlertmanagerConfig.mockClear();
+    // Setting defaultHidden will disable visibility check on aria-label
+    // and speed up the test that use `getByRole`.
+    configure({ defaultHidden: true });
+  });
+  afterEach(() => server.resetHandlers());
+  afterAll(() => server.close());
+
+  const selectors = {
+    enableConfiguration: () =>
+      screen.getByLabelText(/Enable Email Notification Configuration \*/i),
+    host: () => screen.getByLabelText(/SMTP Host \*/i),
+    port: () => screen.getByLabelText(/SMTP Port \*/i),
+    enableTls: () => screen.getByLabelText(/Enable SMTP Over TLS/i),
+    authSelect: {
+      label: () => screen.getByLabelText(/SMTP Auth \*/i).parentElement,
+      authClick: () => screen.getByLabelText(/SMTP Auth \*/i),
+      optionNoAuth: () =>
+        screen.getByRole('option', { name: /NO AUTHENTICATION/i }),
+      optionLogin: () => screen.getByRole('option', { name: /LOGIN/i }),
+      optionCramMd5: () => screen.getByRole('option', { name: /CRAM-MD5/i }),
+      optionPlain: () => screen.getByRole('option', { name: /PLAIN/i }),
+    },
+    username: () => screen.getByLabelText(/Username \*/i),
+    password: () => screen.getByLabelText(/Password \*/i),
+    identity: () => screen.getByLabelText(/identity \*/i),
+    secret: () => screen.getByLabelText(/secret \*/i),
```

Comment:
nit: Identity  and secret are not capitalized here, as we are using the "i" regex flag i think it is totally fine but just need to ensure it is capitalised in the form

Reply:
Indeed, I will remove the 'i' and capitalized it.